Phonics/Content Model/PHContent.swift

```diff
@@ -181,6 +207,11 @@ class PHContentManager {
         })
     }
     
+    //phonics sorted according to CSV and chart ordering
+    func allPhonicsSorted() -> [Sound] {
```

Comment:
Change this to a dynamic variable (e.g. `var allPhonicsSorted [Sound]: { ... }`)